htcanalyze/main.py

```diff
@@ -21,15 +21,22 @@
 
 import configargparse
 
+# import own classes
 from htcanalyze.htcanalyze import HTCAnalyze, raise_value_error
 from htcanalyze.logvalidator import LogValidator
+# import for colorful output
 from rich import print as rprint, box
 from rich.progress import Table
 
-# typing identities
+
+# after analyzing, the information for each log file is saved
+# in it's on dictionary
```

Comment:
`it's on` -> `its own`

What does `log_inf_list` list stand for? Is there a better name that includes the meaning of: "after analyzing, the information for each log file is saved in its own dictionary"?

Reply:
I would suggest brevity when it comes to comments. I think typing identities is perfectly clear, although I do think this section is not needed.

Basically, `List` is a class object from the `typing` module, which can be used for type declarations in functions. Defining `List[dict]` means that the variable (in case of the function, its input or output) is a list of dictionaries. Rather than covertly renaming it to `log_inf_list`, I suggest using `List[dict]`.

I would only suggest renaming if you have particularly long type checks. Such as `Union[List[dict], List[List[str]]]` for very particular cases.

Reply:
Thanks for making this clear. In this case it makes it more complicated than it needs to be